src/NetmockResponse.ts

```diff
@@ -95,3 +99,7 @@ export function getCurrentNetmockReplyTrace() {
 export function clearCurrentNetmockReplyTrace() {
   currentNetmockReplyTrace = undefined;
 }
+
+export function isInstanceOfNetmockResponse(res: any) {
+  return res?.__isNetmockResponse;
```

Comment:
we used to check it using `instanceof` but it doesn't work with reset modules, because jest somehow creates two different classes and the check fails.